incubator/buzzfeed-sso/templates/auth-deployment.yaml

```diff
@@ -95,8 +95,10 @@ spec:
               value: {{ $authDomain }}
             # Provider variables
           {{- with .Values.provider.google }}
+          {{- if .adminEmail }}
             - name: GOOGLE_ADMIN_EMAIL
               value: {{ .adminEmail | quote }}
+          {{- end }}
```

Comment:
@darioblanco I would also take inside the if the `GOOGLE_SERVICE_ACCOUNT_JSON` since it is useful only if the `adminEmail` is present. Same with the `google-service-account` volumeMounts.

Reply:
(the whole point of making `adminEmail` optional is to avoid having to generate a service account when one does not want to use group based access control)

Reply:
Updated. If no admin email is given, the google provider will only work with the client id and client secret stored in `$googleSecret`